boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml

```diff
@@ -36,7 +36,17 @@
                 <artifactId>javax.transaction-api</artifactId>
                 <version>1.2</version>
             </dependency>
-		</dependencies>
+			<dependency>
+				<groupId>javax.enterprise</groupId>
+	            <artifactId>cdi-api</artifactId>
+				<version>2.0</version>
+			</dependency>
+			<dependency>
+    			<groupId>javax.json</groupId>
+    			<artifactId>javax.json-api</artifactId>
+    			<version>1.1</version>
+			</dependency>
+		</dependencies>            
     </dependencyManagement>
  
```

Comment:
Adding CDI, JSONP to the EE 8 BOMs probably made sense back when this PR started, but seems kind of like an odd fit given what we've done in the meantime since now these are defined in the MP2.x BOM.  

So the value add of this now would be in non-MP CDI apps with EE 8 as the "platform" (BOM).    That's important, but it probably is worth adding some IT.   Possibly also worth doing EE 7 at the same time since there's a good chance we'd do a shortcut and just use the same test (not to discourage someone from differentiating though if that can be done easily enough).

Reply:
i am going to remove these from the ee8 bom and do this work under a new PR